.github/actions/run-snowplow-micro/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Run Snowplow micro
+runs:
+  using: "composite"
+  steps:
+    - name: Download Snowplow micro JAR
```

Comment:
Would it be possible to cache it, like it was with CircleCI?

Reply:
This entire step, including the download, takes 6 seconds (maybe because it's all within the same GitHub infrastructure). If this proves to be flaky in the (near) future, I'll follow it up with some caching.